videos/Video.js

```diff
@@ -50,6 +61,29 @@ class Video {
             log("Video URL is null - ad.");
             this.isShort = true;
         }
+        log('video ' + this.videoId + " associated with date " + this.fuzzyDate)
+        if (this.fuzzyDate != null) {
+            this.isOlder = false;
+            log(this.fuzzyDate)
+            if (this.fuzzyDate.includes("month") || this.fuzzyDate.includes("year")) {
+                this.isOlder = true;
+            }
+            else if (this.fuzzyDate.includes("weeks") && hideOlderCutoff != "1 Month") {
+                this.isOlder = true;
+            }
+            else if (this.fuzzyDate.includes("day")) {
+                if (hideOlderCutoff == "Today") {
+                    this.isOlder = true;
+                }
+                else if (hideOlderCutoff == "1 Week" && Number(this.fuzzyDate.split(" ")[0]) >= 7) {
```

Comment:
Fails to properly infer dates for streamed videos (the text includes the word "Streamed", eg. "Streamed 2 weeks ago")

Reply:
switched to regex seems to be working now - [98c870f](https://github.com/OsaSoft/youtube-better-subscriptions/pull/191/commits/98c870f97177bdd44d86a7f5d96d3bb5fc97b4ec)